contracts/cw721-marketplace-permissioned/Cargo.toml

```diff
@@ -44,18 +33,19 @@ optimize = """docker run --rm -v "$(pwd)":/code \
 """
 
 [dependencies]
-cosmwasm-std = "~1.5.0"
-cosmwasm-storage = "~1.0.0"
-cw-storage-plus = "0.12"
-cw2 = "0.12"
-cw20 = { version = "0.13.4" }
-cw20-base = "0.13.4"
-cw721-base = "0.13.4"
-cw721 = "0.13.4"
-schemars = "0.8"
-serde = { version = "1.0", default-features = false, features = ["derive"] }
-thiserror = "1.0"
+cosmwasm-std.workspace = true
+cosmwasm-storage.workspace = true
+cw-storage-plus.workspace = true
+cw2.workspace = true
+cw20.workspace = true
+cw20-base.workspace = true
+cw721-base.workspace = true
+cw721.workspace = true
+schemars.workspace = true
+serde.workspace = true
+thiserror.workspace = true
+utils = { path = "../../utils" }
```

Comment:
Are we sure that relative paths work for publishing to Crates.io? I think it might need to be updated to a GitHub path

Reply:
Crates.io pulls from the github repo so relative paths should be fine as long as the paths in question are inside the repo